scripts/generate_site_sidebar/sidebar_template.md.tmpl

```diff
@@ -25,6 +25,11 @@
       - [install-resource-group](reference/cli/live/install-resource-group/)
       - [migrate](reference/cli/live/migrate/)
       - [status](reference/cli/live/status/)
+    - [alpha](reference/cli/alpha/)
+      - [repo](reference/cli/alpha/repo/)
+        - [get](reference/cli/alpha/repo/get/)
+        - [reg](reference/cli/alpha/repo/reg/)
+        - [unreg](reference/cli/alpha/repo/unreg/)
```

Comment:
I find `reg/unreg` unintuitive. should we use `repo add|remove` instead ?

Reply:
I'd recommend collecting all CLI surface questions and addressing them together. Renaming the reg/unreg or add/remove command is a good suggestion but since renames such as this are disruptive (we need to update docs, design doc, tests, ...) I'd recommend doing them at once.

Reply:
Ack.